scripts/install

```diff
@@ -79,9 +79,17 @@ copy_source() {
 }
 
 
+set_goroot_bootstrap() {
+	[ -z "$GOROOT_BOOTSTRAP" ] && export GOROOT_BOOTSTRAP=$GOROOT
+	if [ -z "$GOROOT_BOOTSTRAP" ]; then
+		local go_path=$(which go)
+		export GOROOT_BOOTSTRAP=${go_path%%/bin/go}
+	fi
+}
+
 compile_go() {
 	display_message " * Compiling..."
-	[ -z "$GOROOT_BOOTSTRAP" ] && export GOROOT_BOOTSTRAP=$GOROOT
```

Comment:
I think a better alternative would be to set a local to check if a goroot exists (and if there is no go, do nothing) 

    local goroot=$(go env GOROOT)
    [ -n goroot] && export GOROOT=goroot
    [ -z "$GOROOT_BOOTSTRAP" ] && export GOROOT_BOOTSTRAP=$GOROOT/bin/go